server/src/main/java/org/opensearch/index/query/QueryStringQueryBuilder.java

```diff
@@ -136,7 +135,7 @@ public class QueryStringQueryBuilder extends AbstractQueryBuilder<QueryStringQue
 
     private boolean enablePositionIncrements = DEFAULT_ENABLE_POSITION_INCREMENTS;
 
-    private Fuzziness fuzziness = DEFAULT_FUZZINESS;
+    private Fuzziness fuzziness = null;
```

Comment:
It looks like this change (and the associated `readOptionalWriteable`/`writeOptionalWriteable` changes) are breaking [backward-compatibility tests](https://build.ci.opensearch.org/job/gradle-check/7577/testReport/junit/org.opensearch.backwards/MixedClusterClientYamlTestSuiteIT/test__p0_range_10_basic_Long_range_/), which makes sense. The existing serialization assumes that fuzziness is **not** optional.

I don't think we can change that behavior.

Reply:
> It looks like this change (and the associated `readOptionalWriteable`/`writeOptionalWriteable` changes) are breaking [backward-compatibility tests](https://build.ci.opensearch.org/job/gradle-check/7577/testReport/junit/org.opensearch.backwards/MixedClusterClientYamlTestSuiteIT/test__p0_range_10_basic_Long_range_/), which makes sense. The existing serialization assumes that fuzziness is **not** optional.
> 
> I don't think we can change that behavior.

Yes, the fuzziness field should be optional, but the previous implementation did the opposite. I would create another issue to discuss it I guess...